src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import com.google.api.services.cloudresourcemanager.model.BooleanConstraint;
+import com.google.api.services.cloudresourcemanager.model.Constraint;
+import com.google.api.services.cloudresourcemanager.model.ListConstraint;
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.Objects;
+
+/** A Google Cloud Resource Manager constraint metadata object. */
+public class ConstraintInfo implements Serializable {
+
+  private static final long serialVersionUID = 9148970963697734236L;
+
+  static final Function<Constraint, ConstraintInfo> FROM_PB_FUNCTION =
+      new Function<Constraint, ConstraintInfo>() {
+        @Override
+        public ConstraintInfo apply(Constraint pb) {
+          return ConstraintInfo.fromPb(pb);
+        }
+      };
+  static final Function<ConstraintInfo, Constraint> TO_PB_FUNCTION =
+      new Function<ConstraintInfo, Constraint>() {
+        @Override
+        public Constraint apply(ConstraintInfo constraintInfo) {
+          return constraintInfo.toPb();
+        }
+      };
+
+  private BooleanConstraint booleanConstraint;
+  private String constraintDefault;
+  private String description;
+  private String displayName;
+  private Constraints constraints;
+  private String name;
+  private Integer version;
+
+  /**
+   * A Constraint that allows or disallows a list of string values, which are configured by an
+   * Organization's policy administrator with a Policy.
+   */
+  static class Constraints implements Serializable {
```

Comment:
If this is a singkle value, this should be `Constraint`

Reply:
`Constraints` would be better due to multiple values